business/tls.go

```diff
@@ -45,7 +45,7 @@ func (in *TLSService) MeshWidemTLSStatus(namespaces []string) (models.MTLSStatus
 
 func (in *TLSService) hasMeshPolicyEnabled(namespaces []string) (bool, error) {
 	if len(namespaces) < 1 {
-		return false, fmt.Errorf("can't find MeshPolicies without a namespace")
+		return false, fmt.Errorf("Unable to determine mesh-wide mTLS status without access to a namespace")
```

Comment:
Can we reword from 'a' namespace to either
* 'any namespace' if we only need access to one to determine this
or
* name the namespace that is needed to be accessed for that operation?